app/src/main/java/de/westnordost/streetcomplete/quests/parcel_locker_brand/AddParcelLockerBrand.kt

```diff
@@ -0,0 +1,31 @@
+package de.westnordost.streetcomplete.quests.parcel_locker_brand
+
+import de.westnordost.streetcomplete.R
+import de.westnordost.streetcomplete.data.osm.geometry.ElementGeometry
+import de.westnordost.streetcomplete.data.osm.mapdata.Element
+import de.westnordost.streetcomplete.data.osm.mapdata.MapDataWithGeometry
+import de.westnordost.streetcomplete.data.osm.mapdata.filter
+import de.westnordost.streetcomplete.data.osm.osmquests.OsmFilterQuestType
+import de.westnordost.streetcomplete.data.user.achievements.EditTypeAchievement.CITIZEN
+import de.westnordost.streetcomplete.osm.Tags
+
+class AddParcelLockerBrand : OsmFilterQuestType<String>() {
+
+    override val elementFilter = "nodes with amenity = parcel_locker and !brand"
```

Comment:
maybe we should exclude ones without name tags?

Reply:
I guess you mean "exclude ones *with* name tags", right?

Reply:
yes, definitely

Reply:
Not really relevant for Amazon lockers. They have their own name.

Reply:
For me, it wouldn't make sense for Amazon lockers, for example.

Reply:
Fixed.